venues/ICLR.cc/2018/Conference/python/reviewer-groups.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
```

Comment:
I'm not sure about this script:

- We don't know how many reviewer are going to be per paper, 5?
- We should create the AnonReviewer group in the assignment script like we did in previous conferences.
- If the reason is because the assignment group doesn't work very well then we should fix it.